### ISSUE TITLE
Obsolete image reference `gcr.io/google_samples/gb-frontend:v5` causes failures in "Deploying PHP Guestbook application with Redis" tutorial

### ISSUE THREAD

**Issue by nomastmas** [kubernetes/website#44224]
**This is a Bug Report**

**Problem:**
The image URL `gcr.io/google_samples/gb-frontend:v5` in this [file](https://github.com/kubernetes/website/blob/de58ecd526d9b2e5d0ba36f7354d6f7b72bb4dd6/content/en/examples/application/guestbook/frontend-deployment.yaml#L20) (which sources this [tutorial](https://kubernetes.io/docs/tutorials/stateless-application/guestbook/#creating-the-guestbook-frontend-deployment)) is broken, resulting in `ErrImagePull` errors.

The root issue is that the `gb-frontend` [image](https://console.cloud.google.com/gcr/images/google-samples/GLOBAL/gb-frontend) is no longer tagged with any versions.

<img width="1384" alt="image" src="https://github.com/kubernetes/website/assets/1271544/0541db51-c84e-4484-a0e4-27b47f190aff">

**Proposed Solution:**

Update the image URL to `us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v5` which is what the [GKE](https://cloud.google.com/kubernetes-engine/docs/tutorials/guestbook) tutorial is using.

**Comment by dipesh-rawat** [comment]
_Page related to issue:_ https://kubernetes.io/docs/tutorials/stateless-application/guestbook/
/language en

**Comment by dipesh-rawat** [comment]
This looks similar (possibly duplicate) of issue https://github.com/kubernetes/website/issues/44190 we have received earlier.

**Comment by divya-mohan0209** [comment]
@nomastmas : Thank you for the list!

These are files belonging to the respective localizations and will require a separate PR since we have a different process per localization team.

```
content/ja/docs/concepts/workloads/controllers/replicaset.md
70:    Image:        gcr.io/google_samples/gb-frontend:v3

content/ja/examples/podpreset/replicaset-merged.yaml
13:    image: gcr.io/google_samples/gb-frontend:v3

content/ja/examples/podpreset/replicaset.yaml
20:        image: gcr.io/google_samples/gb-frontend:v3

content/ja/examples/controllers/frontend.yaml
21:        image: gcr.io/google_samples/gb-frontend:v3

content/ru/examples/application/guestbook/frontend-deployment.yaml
20:        image: gcr.io/google_samples/gb-frontend:v5

content/zh-cn/docs/concepts/workloads/controllers/replicaset.md
176:    Image:        gcr.io/google_samples/gb-frontend:v3

content/zh-cn/blog/_posts/2015-12-00-Managing-Kubernetes-Pods-Services-And-Replication-Controllers-With-Puppet.md
99:  frontend_image => 'gcr.io/google_samples/gb-frontend:v3',
112:  frontend_image => 'gcr.io/google_samples/gb-frontend:v3',

content/zh-cn/examples/application/guestbook/frontend-deployment.yaml
20:        image: gcr.io/google_samples/gb-frontend:v5

content/zh-cn/examples/controllers/frontend.yaml
21:        image: gcr.io/google_samples/gb-frontend:v3

content/pt-br/docs/concepts/workloads/controllers/replicaset.md
74:    Image:        gcr.io/google_samples/gb-frontend:v3

content/pt-br/examples/controllers/frontend.yaml
21:        image: gcr.io/google_samples/gb-frontend:v3

content/ko/docs/concepts/workloads/controllers/replicaset.md
99:    Image:        gcr.io/google_samples/gb-frontend:v3

content/ko/examples/application/guestbook/frontend-deployment.yaml
20:        image: gcr.io/google_samples/gb-frontend:v5

content/ko/examples/controllers/frontend.yaml
21:        image: gcr.io/google_samples/gb-frontend:v3

content/id/docs/concepts/workloads/controllers/replicaset.md
71:    Image:      gcr.io/google_samples/gb-frontend:v3

content/id/examples/controllers/frontend.yaml
21:        image: gcr.io/google_samples/gb-frontend:v3

content/fr/docs/concepts/workloads/controllers/replicaset.md
83:    Image:      gcr.io/google_samples/gb-frontend:v3

content/fr/examples/controllers/frontend.yaml
21:        image: gcr.io/google_samples/gb-frontend:v3

content/es/docs/concepts/workloads/controllers/replicaset.md
88:    Image:      gcr.io/google_samples/gb-frontend:v3

content/es/examples/controllers/frontend.yaml
21:        image: gcr.io/google_samples/gb-frontend:v3
```

This is from the blog subproject. Since we have a policy of not maintaining blogs older than a year, this might not require updating unless this is pivotal information.

```
content/en/blog/_posts/2015-12-00-Managing-Kubernetes-Pods-Services-And-Replication-Controllers-With-Puppet.md
53:  frontend_image => 'gcr.io/google_samples/gb-frontend:v3',
```

What I'd recommend is these three files being PR'ed together to change the image in the English website files. 

```
content/en/examples/application/guestbook/frontend-deployment.yaml
20:        image: gcr.io/google_samples/gb-frontend:v5

content/en/examples/controllers/frontend.yaml
21:        image: gcr.io/google_samples/gb-frontend:v3

content/en/docs/concepts/workloads/controllers/replicaset.md
103:    Image:        gcr.io/google_samples/gb-frontend:v3
```

**Comment by aj11anuj** [comment]
Fixed by #44290 
/close
